letsencrypt/DOCS.md

```diff
@@ -43,118 +43,116 @@ dns:
 
 In addition add the fields according to the credentials required by your DNS provider:
 
-
 ```yaml
 propagation_seconds: 60
-azure_config: ''
-cloudflare_email: ''
-cloudflare_api_key: ''
-cloudflare_api_token: ''
-cloudxns_api_key: ''
-cloudxns_secret_key: ''
-digitalocean_token: ''
-directadmin_url: ''
-directadmin_username: ''
-directadmin_password: ''
-dnsimple_token: ''
-dnsmadeeasy_api_key: ''
-dnsmadeeasy_secret_key: ''
-google_creds: ''
-google_domains_access_token: ''
-google_domains_zone: ''
-hetzner_api_token: ''
-gehirn_api_token: ''
-gehirn_api_secret: ''
-linode_key: ''
-linode_version: ''
-luadns_email: ''
-luadns_token: ''
-njalla_token: ''
-nsone_api_key: ''
-ovh_endpoint: ''
-ovh_application_key: ''
-ovh_application_secret: ''
-ovh_consumer_key: ''
-rfc2136_server: ''
-rfc2136_port: ''
-rfc2136_name: ''
-rfc2136_secret: ''
-rfc2136_algorithm: ''
-aws_access_key_id: ''
-aws_secret_access_key: ''
-sakuracloud_api_token: ''
-sakuracloud_api_secret: ''
-netcup_customer_id: ''
-netcup_api_key: ''
-netcup_api_password: ''
-gandi_api_key: ''
-gandi_sharing_id: ''
-transip_username: ''
-transip_api_key: ''
-inwx_username: ''
-inwx_password: ''
-inwx_shared_secret: ''
+azure_config: ""
+cloudflare_email: ""
+cloudflare_api_key: ""
+cloudflare_api_token: ""
+cloudxns_api_key: ""
+cloudxns_secret_key: ""
+digitalocean_token: ""
+directadmin_url: ""
+directadmin_username: ""
+directadmin_password: ""
+dnsimple_token: ""
+dnsmadeeasy_api_key: ""
+dnsmadeeasy_secret_key: ""
+google_creds: ""
+google_domains_access_token: ""
+google_domains_zone: ""
+hetzner_api_token: ""
+gehirn_api_token: ""
+gehirn_api_secret: ""
+linode_key: ""
+linode_version: ""
+luadns_email: ""
+luadns_token: ""
+njalla_token: ""
+nsone_api_key: ""
+ovh_endpoint: ""
+ovh_application_key: ""
+ovh_application_secret: ""
+ovh_consumer_key: ""
+rfc2136_server: ""
+rfc2136_port: ""
+rfc2136_name: ""
+rfc2136_secret: ""
+rfc2136_algorithm: ""
+aws_access_key_id: ""
+aws_secret_access_key: ""
+sakuracloud_api_token: ""
+sakuracloud_api_secret: ""
+netcup_customer_id: ""
+netcup_api_key: ""
+netcup_api_password: ""
+gandi_api_key: ""
+gandi_sharing_id: ""
+transip_username: ""
+transip_api_key: ""
+inwx_username: ""
+inwx_password: ""
+inwx_shared_secret: ""
```

Comment:
this list should be extended

Reply:
Hi, I have added the two new config options to the list in a new commit